client/landing/gutenboarding/components/plans/plans-button/index.tsx

```diff
@@ -54,7 +55,7 @@ const PlansButton: React.FunctionComponent< Button.ButtonProps > = ( { ...button
 			<Button
 				onClick={ handleButtonClick }
 				label={ __( planLabel ) }
-				className="plans-button"
+				className={ classNames( 'plans-button', { 'user-selected': selectedPlan } ) }
```

Comment:
Can you add `is-` prefix here so that it's clear it's a state indicator, if that makes sense?

Reply:
100%!